scripts/stac/imagery_stac.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Dict, List
+
+from pystac import get_stac_version
+
+from scripts.files.files_helper import get_file_name_from_path, strip_extension
+from scripts.stac.util import checksum  # dont touch this
```

Comment:
Could you add an explaination on why not touching that? and maybe some additional research should be done to clearly understand the issue

Reply:
ha, oops I forgot about this comment, I didn't mean for it to come through, but yes I will update it.